.github/workflows/main.yml

```diff
@@ -79,6 +80,9 @@ jobs:
           mv index.yaml.tmp index.yaml
           popd
 
+          # update the README
+          sed -i "s/version: [[:digit:]].[[:digit:]].[[:digit:]]/version: ${{ env.CHART_VERSION }}/" README.md
```

Comment:
@mattmessinger does CHART_VERSION start with a `v` ?  e.g., `v1.0.8` ?

Reply:
code removes the "v" around line 39

Reply:
indeed.  thx!